about.css

```diff
@@ -0,0 +1,560 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+html,
+body {
+  width: 100%;
+}
+
+.about_page {
+  background-image: url(./images/hero.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  padding-bottom: 50px;
+  height: 90vh;
+}
```

Comment:
- [ ] Your about page on larger window sizes can have a lot of whitespace due to the `vh` value you added for the height. I recommend you change that because it can be really annoying for some users that have larger screens.
 - [ ] The content of your about page first section is centered in the page. Kindly fix the alignment to make it centered.
 - Your design:
 - 
<img width="1034" alt="image" src="https://user-images.githubusercontent.com/61919617/204296396-cebb585a-e387-4781-b98e-672def270755.png">

Reply:
Issues resolved.

- The vh has been reduced for the desktop version.
- Content of hero section is centered. 
Thank you